.github/scripts/check_config_changes_ci.py

```
@@ -0,0 +1,422 @@
+        if header.lower() in body.lower():
+            idx = body.lower().index(header.lower()) + len(header)
+            return body[:idx] + "\n\n" + note + body[idx:]
+ 
+    # 3. Fallback — append
+    return body + "\n\n## Release Notes\n\n" + note
+ 
+ 
+# ── GitHub API ─────────────────────────────────────────────────────────────────
+ 
+def get_pr_body() -> str:
+    r = requests.get(f"{BASE_URL}/repos/{REPO}/pulls/{PR_NUMBER}", headers=HEADERS)
```

Ruff (0.15.9)

[error] Probable use of `requests` call without timeout

(S113)